### ISSUE TITLE
Error in SchurComplementSolvers.jl

### ISSUE THREAD

**Issue by oriolcg** [gridap/GridapSolvers.jl#41]
I think there is an error in this line: https://github.com/gridap/GridapSolvers.jl/blob/0e626022447647d476d7d3f2f44c7b6e09757136/src/LinearSolvers/SchurComplementSolvers.jl#L65

To my understanding it should be something like:

```
copy!(bp,y_p); mul!(bp,C,x_u,-1.0,1.0)  # bp  = y_p - C*(A^-1 y_u)
```

I was having wrong results in my application and this modification gives the good solution.

**Comment by JordiManyer** [comment]
Yes, that looks like a bug. I'll have a look. In any case that solver is due a rather big revision soon.
